hash/msx2_flop.xml

```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
 		</part>
 	</software>
 
-	<software name="nvmg9608">
-		<description>NV Magazine 1996-08 (Japan)</description>
-		<year>1996</year>
-		<publisher>&lt;doujin&gt;</publisher>
-		<info name="developer" value="Syntax" />
+	<software name="nvh1">
+		<description>NV Hokkaido Vol. 1 (Japan)</description>
+		<year>1995</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NV北海道#1"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv hokkaido vol 1 - syntax.dsk" size="737280" crc="76dc41e2" sha1="a1d472a4872490c56db506c615879ae750da7356"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg1">
+		<description>NV Magazine 1 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン1号"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 1 - syntax.dsk" size="737280" crc="7fe2b042" sha1="397ac3feed7da3c58145f362aea3ffbb001f9860"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg2">
+		<description>NV Magazine 2 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン2号"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 2 - syntax.dsk" size="737280" crc="656259a0" sha1="2b707101e0afd063f17eb67ae6e5d54772a81890"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg3">
+		<description>NV Magazine #3 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン#3"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 3 - syntax.dsk" size="737280" crc="6c142379" sha1="f15b8be0c72ea24f9cbf3ee09311a4f52cb60b48"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg4">
+		<description>NV Magazine 4 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン#3"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 4 - syntax.dsk" size="737280" crc="ccf812c5" sha1="ee5f3d652e33a7d15d6f16134076e0bb9b8a9c93"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Description and Japanese title don’t match – Japanese title is a duplicate of `nvmg3`.

Reply:
Fixed